easybuild/easyconfigs/p/pydicom/pydicom-1.2.2-GCCcore-8.2.0.eb

```diff
@@ -0,0 +1,23 @@
+easyblock = 'PythonPackage'
+
+name = 'pydicom'
+version = '1.2.2'
+
+github_account = 'pydicom'
+homepage = 'https://github.com/%(github_account)s/%(name)s'
+description = "Pure python package for DICOM medical file reading and writing."
+
+toolchain = {'name': 'GCCcore', 'version': '8.2.0'}
+
+source_urls = [GITHUB_SOURCE]
+sources = ['v%(version)s.tar.gz']
+checksums = ['f3080249aea2243ded81c3019f0af6e06b7b187d624d06ec89d9e3fac58a204c']
+
+multi_deps = {'Python': ['3.7.2', '2.7.15']}
+
+builddependencies = [('binutils', '2.31.1')]
+
```

Comment:
@lexming can you confirm that the `modulename` has changed since the previous EB version?

Reply:
@verdurin It's indeed `pydicom` now, see https://pydicom.github.io/pydicom/stable/getting_started.html